public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Answer.js

```diff
@@ -0,0 +1,48 @@
+import React, { PropTypes } from 'react';
+import {ManagedForm, ManagedField} from '../../../ManagedEditor';
+import FormFieldTextInput from '../../../FormFields/FormFieldTextInput';
+import SearchSelectBox from '../../../FormFields/SearchFields/SearchSelectBox';
+
+export class StoryQuestionsAnswer extends React.Component {
+  static propTypes = {
+    fieldLabel: PropTypes.string,
+    fieldName: PropTypes.string,
+    fieldValue: PropTypes.shape({
+      answerId: PropTypes.string,
+      answerType: PropTypes.string
+    }),
+    fieldPlaceholder: PropTypes.string,
+    onUpdateField: PropTypes.func,
+    onFormErrorsUpdate: PropTypes.func
+  };
+
+  onUpdate = (answer) => {
+    const answerWithType = Object.assign({}, answer, {
```

Comment:
Just curious: if the field is required, why do you need to override `answerType`?

Reply:
This is effectively setting a default value if it hasn't been set. There's probably a nicer way to do this...